lib/web/app.js

```diff
@@ -57,6 +58,10 @@ module.exports.mount = app => {
 	app.use(hostMatching);
 	app.use(authentication.withUser);
 
+	app.get('/manifest.webmanifest', pwa.getManifest);
```

Comment:
```suggestion
	app.get('/api/v0/web-manifest', pwa.getManifest);
```

You might also have to set `content-type` headers

Reply:
No longer needed 😁